modules/nlu/src/backend/module-lifecycle/on-bot-mount.ts

```diff
@@ -39,8 +39,9 @@ export function getOnBotMount(state: NLUState) {
           return
         }
 
-        const intentDefs = await getIntents(ghost)
-        const entityDefs = await entityService.getCustomEntities()
+        const api = makeApi({ axios: axiosForBot })
```

Comment:
Can you make a helper method or something to avoid repeating the axiosForBot & client creation everywhere?